gear-programs/vft-gateway/src/wasm/tests/vft_gateway.rs

```diff
@@ -217,43 +218,29 @@ fn test_transfer_vara_to_eth_insufficient_balance() {
     assert_eq!(balance, amount); // Balance should remain unchanged
 }
 
-#[ignore]
-#[tokio::test]
-async fn test_transfer_fails_due_to_gas_depletion_after_bridge_reply() {
+#[test]
+fn test_mint_tokens_from_eth_client() {
```

Comment:
Please also add a test that when `mint_tokens` is called from arbitrary address it will not mint any tokens